README.md

```diff
@@ -87,6 +94,39 @@ example if the repodata has a mix of v1 and v2 ModuleStream objects, the
 index will contain only v2 objects (with the v1 objects automatically
 upgraded internally).
 
+Now, we can start operating on the retrieved data. This guide will
+give only a brief overview of the most common operations. See the API
+specification for a full list of information that can be retrieved.
+
+## Discover the default stream for a particular module.
+## C
+```C
+ module =  modulemd_module_index_get_module(merged_index,'modulename');
+ defaults = modulemd_module_get_defaults (module);
+ printf ("Default stream for modulename is %s", 
+       modulemd_defaults_get_module_name (defaults));
+  ```
+ 
+ ## Get the list of RPMs defining the public API for a particular module NSVC
+ ## C
+```C
+module =  modulemd_module_index_get_module(merged_index,'modulename');
+stream = modulemd_module_get_stream_by_NSVC(module, 'modulestream', 1, 'deadbeef');
+api_list = stream.get_rpm_api() //change this
```

Comment:
This hasn't been updated from the Python example.

Reply:
I couldn't find any function in C for this and `stream.get_dependencies()` . Can you point me towards it?

Reply:
[`modulemd_module_stream_v2_get_rpm_api_as_strv()`](https://sgallagh.fedorapeople.org/docs/libmodulemd/2.0/modulemd-2.0-Modulemd.ModuleStreamV2.html#modulemd-module-stream-v2-get-rpm-api-as-strv) and [`modulemd_module_stream_v2_get_dependencies ()`](https://sgallagh.fedorapeople.org/docs/libmodulemd/2.0/modulemd-2.0-Modulemd.ModuleStreamV2.html#modulemd-module-stream-v2-get-dependencies)